interop-matrix.md

```diff
@@ -0,0 +1,83 @@
+# Flatcar inter-operation matrix
+
+This document tracks Flatcar inter-operability across environments.
+
+Support levels:
```

Comment:
"support" level is confusing. And does not visually convey what we're wanting here.
This mnemonic needs to go away entirely.

Instead, just have columns, so folks can visually deduce the same. Like:

| Environment | Full-Feature (release blocker) | Works | Tested | Owner | Reference (e.g. GH issue) | Notes |
|-|:-:|:-:|:-:|-|-|-|
| EC2 | | X | X | @kinvolk/flatcar-maintainers | #107 | ... |
| Azure | | X | X | @kinvolk/flatcar-maintainers |  | ... |
| GCE | X | X | X | @kinvolk/flatcar-maintainers |  |  |
| Digital Ocean (VMs) | X | X | X | @kinvolk/flatcar-maintainers |  |  |
| Equinix Metal | X | X | X | @kinvolk/flatcar-maintainers |  |  |
| ESXi / vSphere | X | X | X | @kinvolk/flatcar-maintainers |  |  |
| Hetzner Cloud | | | X | [no owner] |  |  |
| Vultr VPS  | | | X | [no owner] |  |  |